Test/floats/SpecialValues.bpl

```diff
@@ -21,6 +21,11 @@ procedure foo() returns (r : float24e8) {
   assert DEQ(d,-0e2047f53e11);
   d := -200e2000f53e11;
 
+  d := 0+zero53e11;
+  assert d == 0e0f53e11;
+  d := 0-zero53e11;
+  assert d == -0e0f53e11;
+
```

Comment:
I think it'd better if we add another assertion `DEQ(0+zero53e11,0-zero53e11)` here to test that plus/minus zeros are equal in terms of IEEE standard.